config/sync/system.performance.yml

```diff
@@ -2,9 +2,9 @@ _core:
   default_config_hash: b2cssrj-lOmATIbdehfCqfCFgVR0qCdxxWhwqa2KBVQ
 cache:
   page:
-    max_age: 31536000
+    max_age: 0
```

Comment:
Do we not want a long TTL for cache here? NIDirect has > 0 at https://github.com/dof-dss/nidirect-drupal/blob/development/config/sync/system.performance.yml